docs/managing_deis/using-a-ldap-auth.rst

```diff
@@ -0,0 +1,48 @@
+:title: Using a LDAP Auth
+:description: How to configure LDAP to Authenticate users.
+
+.. _using-a-ldap-auth:
+
+Using a LDAP Auth
+=================
+
+Deis Controller supports Single Sign On access control, for now Deis is able to authenticate using LDAP or Active Directory.
+
+Settings used by LDAP
+---------------------
+
+=========================================           =================================================================================
+setting                                             description
```

Comment:
We can keep the header, but this should be moved over to http://docs.deis.io/en/latest/customizing_deis/controller_settings/